zulipterminal/ui_tools/views.py

```diff
@@ -1034,6 +1038,54 @@ def __init__(self, controller: Any, title: str, *, zt_version: str,
         super().__init__(controller, widgets, 'ABOUT', popup_width, title)
 
 
+class UserInfoView(PopUpView):
+    def __init__(self,
+                 controller: Any,
+                 user_id: int,
+                 title: str) -> None:
+
+        self.display_data: Dict[str, Any] = dict()
+
+        self._fetch_user_data(controller, user_id)
+
+        user_details = [(key, value) for key, value in
+                        self.display_data.items() if key != 'Name']
+        user_view_content = [(self.display_data['Name'], user_details)]
+
+        popup_width, column_widths = self.calculate_table_widths(
+            user_view_content, len(title))
+        widgets = self.make_table_with_categories(user_view_content,
+                                                  column_widths)
+
+        super().__init__(controller, widgets, 'USER_INFO', popup_width, title)
+
+    def _fetch_user_data(self, controller: Any, user_id: int) -> None:
+        # Get user data from model
+        data = controller.model.get_user_info(user_id)
+
+        # Convert the data to presentation form
+        self.display_data['Name'] = data['full_name']
+        if data['email']:
+            self.display_data['Email'] = data['email']
+        if data['date_joined']:
+            self.display_data['Date joined'] = data['date_joined'][:10]
+        if data['timezone']:
+            self.display_data['Timezone'] = data['timezone']
+            self.display_data['Local time'] = datetime.now(
+                pytz.timezone(str(data['timezone']))).strftime("%H:%M")
```

Comment:
This seems related to the data processing (so model-focused?), and also should likely take into account the local preferences for 24h vs am/pm - though the latter need not be centralized as long as we at least add a comment regarding that.